android/Review-content-and-thank.md

```diff
@@ -0,0 +1,25 @@
+# Tutorial for reviewing user's content
+
+## New tutorial
```

Comment:
I don't think this header is needed.
Actually the whole "Screenshots of the original/old tutorial" of Tutorial-for-new-users.md could be removed too, feel free to do it in a separate pull request.